dev_guide/copy_files_to_container.adoc

```diff
@@ -95,15 +95,15 @@ are not in the local directory.
 [[continuous-syncing-on-file-change]]
 == Continuous Syncing on File Change
 
-The `--watch` flag causes the command to monitor the source path for file system
-changes and sync changes each time they occur. With this argument, the command
-will run forever.
+Using the `--watch` option causes the command to monitor the source path for any
+file system changes and syncronizes changes when they occur. With this argument,
+the command will run forever.
 
-Synchronization occurs after a short quiet period is observed to ensure a
+Synchronization occurs after short quiet periods to ensure a
 rapidly changing file system does not result in continuous synchronization
 calls.
 
-When run with `--watch`, the behavior is effectively the same as manually
-invoking `oc rsync` repeatedly, including any arguments normally passed to `oc
-rsync`. Therefore you can control the behavior via the same flags used with
-manual invocations of `oc rsync`, such as `--delete`.
+When using the `--watch` option, the behavior is effectively the same as
+manually invoking `oc rsync` repeatedly, including any arguments normally passed
+to `oc rsync`. Therefore you can control the behavior via the same flags used
```

Comment:
^ I almost wrote that! But then I second-guessed myself because @bfallonf is :pencil2: captain grammar :memo: and tends to be more correcterer than me.